webapp/package.json

```diff
@@ -7,11 +7,11 @@
     "url": "https://github.com/web-platform-tests/wpt.fyi.git"
   },
   "devDependencies": {
-    "@browser-logos/chrome": "1.0.18",
-    "@browser-logos/chrome-beta": "2.1.10",
-    "@browser-logos/chrome-canary": "1.1.13",
-    "@browser-logos/chrome-dev": "2.1.11",
-    "@browser-logos/chromium": "1.1.5",
+    "@browser-logos/chrome": "2.0.0",
+    "@browser-logos/chrome-beta": "2.0.0",
```

Comment:
@argyleink You should keep the original version for `chrome-beta` and `chrome-dev` as that is the latest version (when they will be updated as the rest those packages will be v3).

---

Note: For what changed, see the package's `CHANGELOG.md` file (e.g.: for [Chrome](https://github.com/alrra/browser-logos/blob/bb02a4d6df6d0261e12a8681b4731ffbbd8770da/src/chrome/CHANGELOG.md#changelog)).

Reply:
ty! will update!